tests/SeekerTest.php

```diff
@@ -28,4 +34,36 @@ public function test_seeker_global_search()
         $response->assertJsonPath('data.1.meta.route', 'authors.show');
         $response->assertJsonPath('data.1.meta.group', 'authors');
     }
+
+    public function test_seeker_global_search_with_prefix()
+    {
+        $post = Post::factory()->create(['title' => 'Hi post']);
+        $author = User::factory()->create(['name' => 'Hi author']);
+
+        Route::prefix('admin')->group(function () {
+            Route::formation('posts', PostFormation::class);
+            Route::formation('authors', AuthorFormation::class);
+
+            Route::seeker('search', [
+                PostFormation::class,
+                AuthorFormation::class,
+            ]);
+        });
+
+        $response = $this->get('admin/search?term=hi');
+
+        $response->assertJsonCount(2, 'data.0.data.0');
+        $response->assertJsonPath('data.0.data.0.value', $post->id);
+        $response->assertJsonPath('data.0.data.0.display', 'Hi post');
+
+        $response->assertJsonCount(2, 'data.1.data.0');
+        $response->assertJsonPath('data.1.data.0.value', $author->id);
+        $response->assertJsonPath('data.1.data.0.display', 'Hi author');
+
+        $response->assertJsonPath('data.0.meta.route', 'admin.posts.show');
+        $response->assertJsonPath('data.0.meta.group', 'admin.posts');
```

Comment:
The group is the resource, the [vue component](https://github.com/headlesslaravel/craniums/blob/main/src/Elements/Seeker.vue#L17-L19) displays it in the results

So we don't need it prefixed here, just the route
```php
$response->assertJsonPath('data.0.meta.group', 'posts');
```
Lets change `group` to `resource` also since that makes it more clear